WORKSPACE

```diff
@@ -6,20 +6,20 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 http_archive(
     name = "io_bazel_rules_go",
+    sha256 = "dbde93957f2a96e6c97edb74e29107b7de1b2b57608b8304401e10326e02c66d",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.22.2/rules_go-v0.22.2.tar.gz",
```

Comment:
any reason not to bump this to the latest v0.23.1 ? I see [architecture related improvements in v0.23.0](https://github.com/bazelbuild/rules_go/releases/tag/v0.23.0)

Reply:
Output paths change in v0.23.0:
https://github.com/bazelbuild/rules_go/releases/tag/v0.23.0
> Output file paths no longer include extra configuration data. For example, //tests/core/go_binary:hello creates the file bazel-bin/tests/core/go_binary/hello_/hello. Note that output file paths are still not stable and should not be depended on unless the "out" attribute is set explicitly (for rules that support it).

Reply:
Looking at every use of `go_binary` we either reference it immediately in a `container_image` or don't do anything with it, in which case our make targets copy them elsewhere, so its possibly only the Makefile would need updating.

we can do that version bump and test it out in a separate PR 👍